concourse/setup.py

```diff
@@ -166,16 +166,15 @@ def create_instance_specific_helm_values(concourse_cfg: ConcourseConfig):
 
 
 def deploy_concourse_landscape(
-        config_dir: str,
+        config_factory: ConfigFactory,
```

Comment:
firstly: would it not suffice to pass-in the required cfg_set (in that case, the cfg_name would not be required)